README.md

```diff
@@ -24,9 +26,20 @@ It is meant for discussion, prototype specification and implementation of a prop
 
 * See the [modified spec](https://webassembly.github.io/reference-types/core/) for details.
```

Comment:
Should we delete `/core/` here, to be consistent with the multi-value spec below?